partner_ref_unique/models/res_partner.py

```diff
@@ -29,3 +30,16 @@ def _check_ref(self):
                         _("This reference is equal to partner '%s'")
                         % other[0].display_name
                     )
+
+    @api.model
+    def _name_search(self, name="", args=None, operator="ilike", limit=100):
+        """Allow searching by ref by default."""
+        if name and operator in ["=", "ilike", "=ilike", "like", "=like"]:
+            args = (
+                expression.OR([[("ref", operator, name)], args])
+                if args
+                else [("ref", operator, name)]
```

Comment:
What is returning `expression.OR([[("ref", operator, name)], args])` if args is `[]`?

(I think it is returning the same than `else` part)

Reply:
Could you add a unittesting using `args` as `None`